subprojects/docs/src/docs/release/notes.md

```diff
@@ -34,22 +34,28 @@ For Java, Groovy, Kotlin and Android compatibility, see the [full compatibility
+Gradle has [an advanced dependency cache](userguide/dependency_resolution.html#sec:dependency_cache) which helps reduce downloads of dependencies to a minimum, inside and across build invocations.
+
+With Gradle 6.1, the dependency cache can now be relocated to another directory or host, without the need to preserve absolute paths, for dependencies cached by Gradle 6.1 and later.
+When moved to a new location or seeded into a host image, builds will be able to leverage the dependency cache and will not need to access the network to download artifacts or metadata if they are already cached. 
 
 Note that the cache should be primed and consumed by same Gradle version for maximum effect.
 See [the documentation](userguide/dependency_resolution.html#sub:cache_copy) for more details. 
 
-This is just one step along the way to help organizations using ephemeral CI agents to reduce the overhead of downloading dependencies during the build.
-
 <a name="compilation-order"></a>
-## Defining compilation order between Groovy, Scala and Java
+## Defining compilation order between JVM languages
 
-Previously, the relationship between Java, Groovy and Scala compilation was hardcoded with explicit task dependencies in the same project.
-Gradle assumed Groovy and Scala compilation would always depend on Java compilation. That is, `compileGroovy` and `compileScala` would directly depend on the output from `compileJava`.
+Combining multiple JVM languages in a single project used to have limitations around which language sources could depend on which other language.
+For example it was quite complicated to make Kotlin code depend on Groovy code because explicit task dependencies were declared by the plugins.
+In addition, Gradle assumed Groovy and Scala compilation would always depend on Java compilation.
 
-These task dependencies have been remodelled using [directory properties](userguide/lazy_configuration.html#).  The relationship between compilation tasks is expressed in the task's classpath. Removing a directory property from the classpath also removes the corresponding task dependency.  This can be used to change the relationship between Java, Groovy and Scala compilation tasks.
+These task dependencies have been remodelled using [directory properties](userguide/lazy_configuration.html#).
+The relationship between compilation tasks is expressed in the task's classpath.
+Removing a directory property from the classpath also removes the corresponding task dependency.
+This can be used to change the relationship between Java, Groovy and Scala compilation tasks.
 
-For example, when combining Groovy and Kotlin in the same project, it was previously difficult to make Kotlin classes depend on Groovy classes.
+Back to our example, when combining Groovy and Kotlin in the same project, you can make Kotlin classes depend on Groovy classes like this:
```

Comment:
```suggestion
For example, when combining Groovy and Kotlin in the same project, you can make Kotlin classes depend on Groovy classes like this:
```

_Back to_ reads strange for me as no concrete example was introduced before.

Reply:
I mention Groovy and Kotlin in the first paragraph, as something hard to do. That's the reference. But if you are not getting it, then it does not work.
Will think some more about it.

Reply:
Changed